linode_api4/objects/vpc.py

```diff
@@ -100,7 +100,7 @@ def subnet_create(
         return d
 
     @property
-    def ips(self, *filters) -> PaginatedList:
+    def ips(self) -> PaginatedList:
```

Comment:
This isn't directly related to this change but it was causing the linter to fail. This shouldn't be a breaking change since arguments could already not be passed into ips 👍